src/modules/mavlink/mavlink_mission.cpp

```diff
@@ -86,8 +86,9 @@ MavlinkMissionManager::MavlinkMissionManager(Mavlink *mavlink) : MavlinkStream(m
 	_offboard_mission_sub(-1),
 	_mission_result_sub(-1),
 	_offboard_mission_pub(nullptr),
-	_slow_rate_limiter(_interval / 5.0f),
-	_verbose(false)
+	_slow_rate_limiter(100 * 1000), // Rate limit sending of the current WP sequence to 10 Hz
```

Comment:
This is not desirable as some links will get heartbeat timeouts if you fill them with mission items while other fast links will take many seconds for a couple hundred mission items even though they could transfer much faster.